pkg/clustertree/cluster-manager/controllers/pv/leaf_pv_controller.go

```diff
@@ -211,13 +212,13 @@ func filterPV(pv *v1.PersistentVolume, nodeName string) {
 		mfs := v.MatchFields
 		mes := v.MatchExpressions
 		for k, val := range v.MatchFields {
-			if val.Key == utils.NodeHostnameValue || val.Key == utils.NodeHostnameValueBeta {
+			if val.Key == utils.NodeHostnameValue || val.Key == utils.NodeHostnameValueBeta || strings.Contains(val.Key, utils.NodeName) {
 				val.Values = []string{nodeName}
```

Comment:
Can `strings.Contains(val.Key, utils.NodeName)` be changed to a more explicit matching method?